packages/atomic/src/components/atomic-load-more-results/atomic-load-more-results.tsx

```diff
@@ -0,0 +1,131 @@
+import {
+  QuerySummary,
+  QuerySummaryState,
+  buildQuerySummary,
+  ResultList,
+  ResultListState,
+  buildResultList,
+} from '@coveo/headless';
+import {Component, h, State} from '@stencil/core';
+import {
+  Bindings,
+  BindStateToController,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+import {createRipple} from '../../utils/ripple';
+
+/**
+ * The `atomic-load-more-results` component allows the user to load additional results if more are available.
+ *
+ * @part container - The container of the component.
+ * @part showing-results - The summary displaying which results are shown and how many are available.
+ * @part highlight - The highlighted number of results displayed and number of results available.
+ * @part progress-bar - The progress bar displaying a percentage of results shown over the total number of results available.
+ * @part load-more-results-button - The "Load more results" button.
+ *
+ * @part ripple - The ripple effect of the component's interactive elements.
+ */
+@Component({
+  tag: 'atomic-load-more-results',
+  styleUrl: 'atomic-load-more-results.pcss',
+  shadow: true,
+})
+export class AtomicLoadMoreResults {
+  @InitializeBindings() public bindings!: Bindings;
+  public querySummary!: QuerySummary;
+  public resultList!: ResultList;
+
+  @BindStateToController('querySummary')
+  @State()
+  private querySummaryState!: QuerySummaryState;
+  @BindStateToController('resultList')
+  @State()
+  private resultListState!: ResultListState;
+  @State() public error!: Error;
+
+  public initialize() {
+    this.querySummary = buildQuerySummary(this.bindings.engine);
+    this.resultList = buildResultList(this.bindings.engine, {
+      options: {
+        fieldsToInclude: [],
+      },
+    });
+  }
+
+  private wrapHighlight(content: string) {
+    return `<span class="font-bold text-on-background" part="highlight">${content}</span>`;
+  }
+
+  private get showingResultsOptions() {
+    const locales = this.bindings.i18n.languages;
+    return {
+      interpolation: {escapeValue: false},
+      last: this.wrapHighlight(
+        this.querySummaryState.lastResult.toLocaleString(locales)
+      ),
+      total: this.wrapHighlight(
+        this.querySummaryState.total.toLocaleString(locales)
+      ),
+    };
+  }
+
+  private renderShowingResults() {
+    const content = this.querySummaryState.hasResults
+      ? this.bindings.i18n.t(
+          'showing-results-of-load-more',
+          this.showingResultsOptions
+        )
+      : '';
+
+    return (
+      <div
+        class="my-2 text-lg text-neutral-dark"
+        part="showing-results"
+        innerHTML={content}
```

Comment:
Perhaps we could use `dompurify`'s `sanitize` method wherever `innerHTML` is set.